.lycheeignore

```diff
@@ -0,0 +1 @@
+blob/[0-9]+\.[0-9]+\.[0-9]+/doc/.*
```

Comment:
probably not needed, if you use absolute urls in the Readme

Reply:
See next push (deleted the file)